PyRoute/Star.py

```diff
@@ -97,17 +100,14 @@ class Star(object):
 """
     starline = re.compile(''.join([line.rstrip('\n') for line in regex]))
 
-    __slots__ = '__dict__', '_hash', '_key', 'index', 'zone', 'tradeCode', 'wtn', 'alg_code', 'x', 'y', 'z'
+    __slots__ = '__dict__', '_hash', '_key', 'index', 'zone', 'tradeCode', 'wtn', 'alg_code', 'hex'
 
     def __init__(self):
         self.logger = logging.getLogger('PyRoute.Star')
         self._hash = None
         self._key = None
         self.component = None
         self.index = None
-        self.x = None
-        self.y = None
-        self.z = None
         self.q = None
```

Comment:
You should be able remove the q/r coordinates from here as well. Everything should be referencing the Hex class.

Reply:
That's all ripped out, replaced by properties wrapping the corresponding values in the Hex object